documentation/modules/ova-prerequisites.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: REFERENCE
+[id="ova-prerequisites_{context}"]
+= Open Virtual Appliance (OVA) prerequisites
+
+The following prerequisites apply to Open Virtual Appliance (OVA) file migrations:
+
+* All OVA files are created by VMware vSphere.
+
+[NOTE]
+====
+Migration of OVA files that were not created by VMware vSphere but are compatible with  vSphere might succeed. However, migration of such files is not supported by {project-short}. {project-short} supports only OVA files created by VMware vSphere.
+====
+
+* The OVA files are in one or more folders under an NFS shared directory.
+* The OVA files are stored in in the NFS shared directory in one of the following structures:
+
+** For OVA files in one or more compressed OVF packages that hold all the VM information:
++
+`<nfs_shared_directory>/folder1/vm1.ova` +
+`<nfs_shared_directory>/folder1/vm2.ova`
++
+_or_
++
+`<nfs_shared_directory>/folder1/vm1.ova` +
```

Comment:
This part looks the same as the above part, should it be removed or changed?

Reply:
I made the differences between the options clearer. How does it look now?

Reply:
Ah, I see, I didn't notice the difference..., it's more clear now